ios/RNFetchBlob/RNFetchBlob.m

```diff
@@ -622,7 +622,12 @@ - (NSDictionary *)constantsToExport
 - (UIViewController *) documentInteractionControllerViewControllerForPreview: (UIDocumentInteractionController *) controller
 {
     UIWindow *window = [UIApplication sharedApplication].keyWindow;
-    return window.rootViewController;
+    UIViewController *currentlyPresentedView = [window.rootViewController presentedViewController];
```

Comment:
Do these changes have anything to do with the content-type header bug described in the PR? This seems to be addressing a different bug, could you describe what issue this change fixes and if/how you tested it?

Reply:
Hi @Traviskn 
This issue is something I had a problem with locally, where I tried to open a document from a view controller that was not the root view controller, thus prompting to me an error along the lines of:

Warning: Attempt to present < UIDocumentSomething: 0x1e56e0a0 > on < MyViewController: 0x1ec3e000> whose view is not in the window hierarchy!